src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -694,7 +694,7 @@ class DetailsCard extends React.Component {
                 <Col className={style['fields-column']}>
                   <Grid>
                     <FieldRow label={msg.host()} id={`${idPrefix}-host`}>
-                      { hostName || <NotAvailable tooltip={msg.notAvailableUntilRunning()} id={`${idPrefix}-host-not-available`} /> }
+                      { <span className={style['hostname']} title={hostName}>{hostName}</span> || <NotAvailable tooltip={msg.notAvailableUntilRunning()} id={`${idPrefix}-host-not-available`} /> }
```

Comment:
I know the discussion about tooltip style (or bootstrap-react tooltip vs title) got pushed, but I'd rather see the patternfly-react.  Otherwise the hidden overflow with ellipsis looks good.

@gregsheremeta - `patternfly-react` technically uses `react-bootstrap` which re-implements the bootstrap components.  I'm not sure the comment about memory leaks like in webadmin is still relevant.  For example: [Tooltip.js](https://github.com/react-bootstrap/react-bootstrap/blob/v0.32.4/src/Tooltip.js) is exported as the patternfly-react Tooltip.